release-tools/prow.sh

```diff
@@ -86,7 +86,7 @@ configvar CSI_PROW_BUILD_PLATFORMS "linux amd64 amd64; linux ppc64le ppc64le -pp
 # which is disabled with GOFLAGS=-mod=vendor).
 configvar GOFLAGS_VENDOR "$( [ -d vendor ] && echo '-mod=vendor' )" "Go flags for using the vendor directory"
 
-configvar CSI_PROW_GO_VERSION_BUILD "1.18" "Go version for building the component" # depends on component's source code
+configvar CSI_PROW_GO_VERSION_BUILD "1.19" "Go version for building the component" # depends on component's source code
```

Comment:
You can't edit files in [release-tools/prow.sh](https://github.com/kubernetes-csi/csi-test/pull/421/files#diff-a4f8e29dad5f252294abb2ef2e93f8325eb9b9d4cd680b7577437a4aa88321fb). If you want 1.19, update the whole subtree using this process https://github.com/kubernetes-csi/csi-release-tools#sharing-and-updating

Reply:
Got it, opened https://github.com/kubernetes-csi/csi-test/pull/431 to fix this